### PR TITLE
(Feature) Allow min cap manage screen

### LINKED ISSUE #930: (Feature) Add ability to management page to update contribution global min cap
**Problem**: there is no way to change global min cap contribution after deployment process. Auth-os gives us such possibility before crowdsale is started.

**Solution**: add global min cap input to manage page. It could be modifiable any time before crowdsale is started regardless of the value of "allow modifying" flag

### PR DESCRIPTION
**Pull request by fernandomg**
Closes #930 

I wasn't able to update `minCap` after crowdsale creation.

I tried with and without setting an initial minCap value in the crowdsale creation process.

Please @vbaranov review

---

Besides, @dennis00010011b `manage` screen layout has slightly changed, so it is possible that e2e test fails.

### CODE REVIEW

**Comment by dennis00010011b** [comment]
4. Error message if   ```globalMinCap === "" ```

screen shot 2018-06-05 at 16 40 03

**Comment by dennis00010011b** [comment]
5. There is no validation  if minCap > totalSupply

File: src/components/manage/ManageDutchAuctionBlock.js [hunk 1]
```diff
@@ -30,22 +31,20 @@ export const ManageDutchAuctionBlock = ({
...
     {fields.map((name, index) => {
       const currentTier = fields.value[index]
-      let { walletAddress } = currentTier
-      const { endTime: initialEndTime, whitelistEnabled } = fields.initial[index]
+      const { startTime: initialStartTime, endTime: initialEndTime, whitelistEnabled } = fields.initial[index]
 
+      const tierHasStarted = !isDateLaterThan()(dateToTimestamp(initialStartTime))(Date.now())
       const tierHasEnded = !isDateLaterThan()(dateToTimestamp(initialEndTime))(Date.now())
-      const canEditWhiteList = canEditTiers && !tierHasEnded
+      const canEditWhiteList = canEditTiers && !tierHasEnded && !tierHasStarted
```
[the remark this answers is not in the document]
Comment (fernandomg) [reply src/components/manage/ManageDutchAuctionBlock.js]: fixed

**Comment by fernandomg** [comment]
@dennis00010011b @vbaranov fixes done, thanks!

Still cannot update minCap, did you guys were able to do it?

**Comment by dennis00010011b** [comment]
@fernandomg  There is still issue, can't add whitelist from manage page. Decimals =18
screen shot 2018-06-06 at 08 54 31

**Comment by dennis00010011b** [comment]
2. There is no validation if minCap >totalSupply
3. In case if all tiers are whitelisted field minCap should be always disabled
4. minCap enabled if crowdsale not modifiable

Yes, I can't update minCap too

**Comment by fernandomg** [comment]
> In case if all tiers are whitelisted field minCap should be always disabled

fixed

> can't add whitelist from manage page. Decimals =18

fixed

---

> There is no validation if minCap >totalSupply

If that's the case, the crowdsale creation process is wrong as well, because if you enable whitelists for only one tier, minCap is disabled and set to zero. 

> minCap enabled if crowdsale not modifiable

Thinking a bit further, rule should be: if all tiers without whitelist enabled are not modifiable, then minCap should not be modifiable either.

Please @dennis00010011b @vbaranov confirm the last two items.

**Comment by dennis00010011b** [comment]
@fernandomg 
There is still issue:  minCap field disabled even if tier modifiable, no whitelisted

**Comment by vbaranov** [comment]
@dennis00010011b @fernandomg 

I guess the rule on the management page could be the same as in step 3 of deployment process:

global min cap is disabled if any of a tier is whitelisted. If none of the tiers is whitelisted, global min cap is enabled and should be less than total crowdsale supply.

**Comment by fernandomg** [comment]
@dennis00010011b @vbaranov 

`minCap` is editable only if there's no whitelist, and it cannot be greater than the minimum supply for the tiers.

**Comment by dennis00010011b** [comment]
@fernandomg 
minCap is editable before start if crowdsale isn't modifiable

screen shot 2018-06-07 at 08 05 33

**Comment by fernandomg** [comment]
> minCap is editable before start if crowdsale isn't modifiable

@dennis00010011b Fixed, so if there's at least one editable tier `minCap` will be editable as well.

**Pull request merged**